openedx/core/djangoapps/ccxcon/api.py

```diff
@@ -16,9 +16,9 @@
 )
 
 from lms.djangoapps.courseware.courses import get_course_by_id
-from lms.djangoapps.instructor.access import list_with_level
 from openedx.core.djangoapps.models.course_details import CourseDetails
 from student.models import anonymous_id_for_user
+from student.roles import CourseInstructorRole
```

Comment:
See [Jenkins](https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/33259/testReport/junit/openedx.core.djangoapps.ccxcon.tests.test_signals/CCXConSignalTestCase/test_course_published_ccxcon_call/) for how this circular dependency was occurring. The correct fix here is to not access anything under `lms.*` from `openedx/*`, correct?